core/src/test/java/org/elasticsearch/action/delete/DeleteResponseTests.java

```diff
@@ -40,18 +40,18 @@
 
     public void testToXContent() {
         {
-            DeleteResponse response = new DeleteResponse(new ShardId("index", "index_uuid", 0), "type", "id", 3, 5, true);
+            DeleteResponse response = new DeleteResponse(new ShardId("index", "index_uuid", 0), "type", "id", 3, 5, true, 0L);
```

Comment:
`0` isn't particularly believable here. Maybe use `timeValueNanos(12345)` or something. That'd show you what the result would look like for a more reasonable number.